arduino/main/src/setVoltage/setVoltage.ino

```diff
@@ -0,0 +1,32 @@
+#include <Wire.h>
+
+#define MAX5434_VREG_COMMAND 17  //write ti volatile register ; wiper position update
+#define MAX5434_NVREG_COMMAND 33  //write to non volatile register ; wiper position unchanged
```

Comment:
Is this used ?

Reply:
2 are used, 2 are not for now. Still good to have them if needed